### DISCUSSION TITLE
Where to place the config.toml file in MacOS?

### DISCUSSION THREAD

**Discussion by bcarney66** [mxpv/podsync#260] [Q&A]
I'm attempting to follow the docs and running this on OS/X MacOS using Docker.  The instructions say: "You need to create a configuration file (for instance config.toml) and specify the list of feeds that you're going to host. Here is an example how configuration might look like:"

Where exactly should this file be saved?

**Comment by ghost** [comment]
As far as I understand you can place it wherever you want because you need to pass its path as argument manually when running docker image.
